GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -800,6 +800,7 @@ private void InitializeComponent()
             // 
             this.closeToolStripMenuItem.Name = "closeToolStripMenuItem";
             this.closeToolStripMenuItem.Size = new System.Drawing.Size(221, 22);
+            this.closeToolStripMenuItem.ShortcutKeys = ((System.Windows.Forms.Keys)((System.Windows.Forms.Keys.Control | System.Windows.Forms.Keys.W)));
```

Comment:
Will it change if changed by the user in Settings?

Reply:
No. But that are not actions where you could changes there shortcuts (and these shortcuts are already displayed in the context menu...)

Reply:
This one is actually configurable, that prompted my original question:

![image](https://user-images.githubusercontent.com/4403806/34673583-d5e940b4-f4d6-11e7-8f97-74b7bc8f436f.png)